niworkflows/interfaces/cifti.py

```diff
@@ -318,7 +318,9 @@ def _create_cifti_image(bold_file, label_file, bold_surfs, annotation_files, tr,
         warnings.warn("Resampling bold volume to match label dimensions")
         bold_img = resample_to_img(bold_img, label_img)
 
-    bold_img = _reorient_image(bold_img, target_img=label_img)
+    # ensure images match HCP orientation (LAS)
+    bold_img = _reorient_image(bold_img, orientation='LAS')
+    label_img = _reorient_image(label_img, orientation='LAS')
```

Comment:
Is this going to be the case for all label files? Should we have a table mapping space to orientation?

Reply:
yes, since the final output should always be in LAS we want to ensure both BOLD / labels are as well prior to setting the IJK voxels

Reply:
I guess my reading of Tim's response was that LAS was an accident of the orientation of a specific template. Are we guaranteed that all templates HCP ever uses will be in LAS?

Reply:
no, we aren't - but that seems excessive for us to worry about. I'd rather mirror the outputs generated from HCP Pipelines, which happen to include the volume in LAS orientation